cliquet/storage/memory.py

```diff
@@ -94,12 +94,19 @@ def apply_filters(self, records, filters):
             COMPARISON.NOT: operator.ne,
             COMPARISON.MIN: operator.ge,
             COMPARISON.GT: operator.gt,
+            COMPARISON.IN: operator.contains,
         }
 
         for record in records:
-            matches = [operators[f.operator](record.get(f.field), f.value)
-                       for f in filters]
-            if all(matches):
+            matches = True
+            for f in filters:
+                left = record.get(f.field)
+                right = f.value
+                if f.operator == COMPARISON.IN:
```

Comment:
I don't like this too much I would rather create the ``COMPARISON.IN: lambda x,y: operator.contains(y, x)``